### ISSUE TITLE
Is project dead?

### ISSUE THREAD

**Issue by firatagdas** [papyros/qml-material#483]
I wanna use this framework in a desktop application for in-house use. I 'm wondering is project dead or gonna continue to develop? Can you make a quick comment?

**Comment by nimaim** [comment]
@iBeliever, any comment? I'd really like to use this for a new desktop application but if it is no longer being maintained, I'll move on to something else. Just curious, I know everyone is busy with other things. Thanks.

**Comment by bentalexanderhaase** [comment]
I'd say more than 7 months no development -> Dead. Hopefully to be revived !

**Comment by andrewrobinson5** [comment]
Entire OS merged with liri, but qml-material is in a usable state. Google the liri project.

**Comment by plfiorini** [comment]
Pretty much what @Cheapskate01 said I just add that qml-material replacement is Fluid https://github.com/lirios/fluid

EDIT: You can read more from here https://liri.io/blog/2017/01/14/fluid-0.9.0.html

**Comment by manhnt9** [comment]
@iBelieve, I would like to know about this too. Please respond, thanks.

**Comment by yasser-sobhy** [comment]
Hi guys, I have an alternative UI framework that I have wrote a couple of
months ago

Unfortunately I didn't publish it yet, but I will publish it today on
Github and send you links

It has many useful features and components and depends on Qt Quick Controls
2

**Comment by vimpostor** [comment]
I don't really think that fluid replaces qml-material. First of all, not even liriOS's own demo works with fluid (I tried compiling and made an issue about this ages ago, but it still isn't fixed), and second fluid does not really strictly follow the Google Material design. Fluid also tries some unnecessary bullshit like being compatible with Microsoft's Universal design (Source: https://liri.io/blog/2017/03/18/fluid-0.10.0.html ). I mean, come on guys, are you really putting two different interface design concepts together at the same time, how would this even remotely work? Just concentrate on Material design already.
For the time being, qml-material is clearly a better choice than fluid.

**Comment by timsueberkrueb** [comment]
@magnus-gross I disagree, but that's fine. No need for being so negative in any case. Nobody stops you from using QML Material despite it being no longer maintained.

**Comment by plfiorini** [comment]
@magnus-gross demo works (for me and others) I added a few instructions for you in the issue just now, besides keep in mind that we follow gitflow so `develop` is a development branch. Universal is a style provided by QtQuick Controls, we are not going to implement any of that, it's not even mentioned in the README.md.

**Comment by vimpostor** [comment]
Ok sry, looks like I was wrong. I will give fluid another try now, but maybe you should include these things in your README in the installation instructions, because it is not a good sign, if a project's own installation instructions do not work (which they don't) - no offense.
Again sry, I may have jumped to conclusions a bit too early.

**Comment by plfiorini** [comment]
The new `fluid-demo.pro` is not documented yet and I'm going to add the env vars details to the README.
It's just that it's normal stuff on Linux and Qt so it got overlooked sorry

**Comment by vimpostor** [comment]
Ok, just once more to set things right:
I just gave Liri's fluid UI another shot and it really went a long way: Everything looks very polished now and I have to completely take back my rant; fluid is clearly better, since it uses QuickControls 2, has about the same spectrum of components available now AND is actively maintained. Sry for the confusion.
I just started a new project (with qml-material in mind), but I will probably use fluid now instead.
